### ISSUE TITLE
Resolution Limitation

### ISSUE THREAD

**Issue by vidda13** [DmitryUlyanov/texture_nets#49]
I see that the pretrained model you provided runs at higher resolutions compared to the models I trained for a few style images. What might be the reason for this?Did you use VGG19 or normalised vgg?

**Comment by besirkurtulmus** [comment]
I don't know the reason, but in the paper he says that he doesn't uses VGG19 at all.

**Comment by vidda13** [comment]
OK, anyway the loss network wont be used once the training is complete. So, How can the FCN size be different ?
